integ-test/rc/esnode-key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
+MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQCWvn+O+rywfgMC
```

Comment:
Should git secrets or something be used to not make this public?

Reply:
Those files are from security plugin demo configuration: https://github.com/opensearch-project/security/tree/main/bwc-test/src/test/resources/security
Probably I can update gradle script to download them

Reply:
If it was useful to change how these are the Security Plugin packaged/published the demo configuration in we'd be interested in issues/pull requests.

Reply:
I download them in 90f28738f